torch/_inductor/lowering.py

```diff
@@ -513,7 +513,7 @@ def squeeze(x, dim=None):
     new_shape = [
         s
         for d, s in enumerate(x.get_size())
-        if not (d in dims and V.graph.sizevars.maybe_guard_equals(s, 1))
+        if not (d in dims and V.graph.sizevars.guard_equals(s, 1))
```

Comment:
Shouldn't this be `evaluate_expr(sympy.Eq(s, 1))` as per https://github.com/pytorch/pytorch/pull/98562#issuecomment-1500299198 since `guard_equals` asserts that it evaluates to True.

Reply:
Yeah I just took it out from Ed's Pr, Im gonna see where this fails.

Reply:
You still need to fix this